README.md

```diff
@@ -1,11 +1,17 @@
 ![Open Data Service (ODS)](https://github.com/jvalue/open-data-service/workflows/Open%20Data%20Service%20(ODS)/badge.svg)
+[![on gitter](https://badges.gitter.im/jvalue-ods/community.svg)](https://gitter.im/jvalue-ods/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 # Open Data Service (ODS)
 
 The Open Data Service (ODS) is an application which can collect data from multiple sources simulataneously, process that data and then offer an improved (or "cleaned") version to its clients.
 *We aim to establish the ODS as **the** go-to place for using Open Data!*
 
+## Getting Help
```

Comment:
Maybe "Contact us" is a better header. 

Here is an alternative text proposal:
_If you have any questions or would like to contact us, you can easily reach us at Gitter or ..._